### ISSUE TITLE
Bug with Truncate messages

### ISSUE THREAD

**Issue by lvca** [orientechnologies/orientdb#6659]
From a user:
``````
We found strange query behaviour with truncate operation.

We have class: `create class Object extends V`

If we have 5 `Objects` and 7 `V`, when executing `truncate class V unsafe`:
- First time: result says that 7 nodes deleted (they are actually deleted)
- Second time: results says that 5 nodes deleted (but they are not deleted)
``````

**Comment by lvca** [comment]
The Truncate is not polymorphic, but this looks like a bug.

**Comment by tglman** [comment]
Hi,

I fixed the wrong count, anyway checking back with the rest of the team i found out that from sql you can use the "POLYMORPHIC" keyword that will allow to truncate as well classes that extends the target.

Regards

**Comment by tglman** [comment]
forgetting, the fix will be released in 2.2.9

**Comment by tglman** [comment]
Hi,

this was fixed and released a while ago, closing

bye
